ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/router-adaptive-virtual-topology.j2

```diff
@@ -0,0 +1,22 @@
+{#
+ Copyright (c) 2023 Arista Networks, Inc.
+ Use of this source code is governed by the Apache License 2.0
+ that can be found in the LICENSE file.
+#}
+{# eos - router-adaptive-virtual-topology #}
+{% if router_adaptive_virtual_topology is arista.avd.defined %}
+!
+router adaptive-virtual-topology
+{%     if router_adaptive_virtual_topology.topology_role is arista.avd.defined %}
+   topology role {{ router_adaptive_virtual_topology.topology_role }}
+{%     endif %}
+{%     if router_adaptive_virtual_topology.region is arista.avd.defined %}
+   region {{ router_adaptive_virtual_topology.region.name }} id {{ router_adaptive_virtual_topology.region.id }}
+{%     endif %}
+{%     if router_adaptive_virtual_topology.zone is arista.avd.defined %}
+   zone {{ router_adaptive_virtual_topology.zone.name }} id {{ router_adaptive_virtual_topology.zone.id }}
+{%     endif %}
+{%     if router_adaptive_virtual_topology.site is arista.avd.defined %}
+   site {{ router_adaptive_virtual_topology.site.name }} id {{ router_adaptive_virtual_topology.site.id }}
+{%     endif %}
```

Comment:
Since schema is still only warning, we should check for each name and id key instead of just the dict.